docs/03-endpoints/api-admin/projects.md

```diff
@@ -813,6 +813,42 @@ Example response:
 }
 ```
 
+### Set Restricted View Settings
+
+The route takes String parameter which sets restricted view size in on of two formats: as an image dimensions or a 
+percentage. The dimensions pattern looks like: `!X,X`, where X is the number representing scaled image dimensions in
+a square, so that the width and height of the returned image are not greater than the requested value.
+Example: `!512,512` means the image's bigger side will be set to 512 pixels, setting the other side respectively to
+image aspect ratio.
+The percentage pattern looks like: `pct:X`, where X is the number between 1-100 representing the percentage the image
+will be scaled to. Example: `pct:1` means the image will be scaled to 1% of the original image size.
+
+Permissions: ProjectAdmin/SystemAdmin
+
+Request definition:
+- `POST /admin/projects/iri/{iri}/RestrictedViewSettings`
```

Comment:
The `GET` api supports _iri_, _shortname_ and _shortcode_.

Should we also support all three possible identifiers for the `POST`?